geoportal/geoportailv3_geoportal/views/wms.py

```diff
@@ -27,6 +31,64 @@ def _check_token(self, token):
             return True
         return False
 
+    def _get_arcgis_token(self, internal_wms, force_renew=False):
```

Comment:
I find this function confusing. Maybe just doing the HTTP requests if the token is expired and not forced to be renewed ?